people/lukas.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "Lukas Forst",
-  "nick": "Lukas",
+  "name": "Lukáš Forst",
+  "nick": "Zajíček",
```

Comment:
... 😄 vazne musime nahajrovat nejakeho studenta, specificky se budou koukat po nekom, kdoo je mladsi nez ja

Reply:
a zajicka pls taky ne 😄  to uz tam dej "umi dobre delat mesic"